src/store/ducks/plans.ts

```diff
@@ -245,6 +256,7 @@ export const extractPlanRecordResponseFromPlanPayload = (
     }
     return planRecordResponse;
   }
+  displayError(new Error(FAILED_TO_EXTRACT_PLAN_RECORD));
```

Comment:
I have mixed feelings about this.

- Does extractPlanRecordResponseFromPlanPayload belong in the dux module - it does not seem to be a reducer function, an action creator or a selector?
- Can we add docstrings to extractPlanRecordResponseFromPlanPayload?

Reply:
- I could move them to the root utils file.